MembershipApp/templates/MembershipApp/base.html

```diff
@@ -35,4 +35,62 @@
 {% endblock %}
 </body>
 
+<footer class="space--sm footer-2">
```

Comment:
1) the html footer should always be inside the body
2) I thought we agreed on moving the footer to its own file and including it at the end of the body using the django include syntax